app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -0,0 +1,139 @@
+angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts'])
+    .config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
+    }])
+    .controller('containerDashboardController', ['$scope', 'ChartsDataMixin', '$http', '$interval', "$location",
+        function($scope, chartsDataMixin, $http, $interval, $location) {
+            dashboard = this;
+
+            $scope.refresh = function() {
+                var url = '/container_dashboard/data';
+                $http.get(url).success(function(response) {
+                    'use strict';
+                    var data = response.data;
+
+                    // stash a ref to the controller object, and the various parent objects
+                    dashboard.chartHeight = chartsDataMixin.dashboardSparklineChartHeight;
+                    dashboard.dashboardHeatmapChartHeight = chartsDataMixin.dashboardHeatmapChartHeight;
+
+                    // Get the container data
+                    dashboard.status_widgets = data.status;
+
+                    // Provider data
+                    dashboard.providers = {
+                        title: data.types.name,
+                        count: data.types.count,
+                        notifications: []
+                    };
+
+                    data.types.types.forEach(function(item) {
+                        dashboard.providers.notifications.push({
+                            iconClass: item.iconClass,
+                            count: item.count,
+                            href:  item.href
+                        })
+                    });
+                    debugger;
+                    });
+
+
+                /*
```

Comment:
lets remove the commented code from here until it should be used to ease review, what do you say? :)